packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -164,6 +195,27 @@ export class AtomicFacet implements InitializableComponent {
     return this.facetState.values.map((value) => this.renderValue(value));
   }
 
+  private renderSearchResults() {
+    return this.facetState.facetSearch.values.map((value) =>
+      this.renderValue({
+        state: 'idle',
```

Comment:
All the search results are idle because we send selected values in the ignoreValues param

Reply:
I'm not sure I understand the ignoreValues param. When building a `specific-facet-search-request`, only the results whose `state` is not `idle` are added to the ignoreValues param?

Reply:
Yes we tell the API not to return values which are already selected, as not to select them twice